geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/test/scala/org/locationtech/geomesa/parquet/ParquetReadWriteTest.scala

```diff
@@ -148,7 +148,7 @@ class ParquetReadWriteTest extends Specification with AllExpectations {
     }
 
     "perform filtering on two week duration" >> {
-      val result = readFile(ECQL.toFilter("dtg BETWEEN '2016-12-131T12:00:00Z' AND '2017-01-01T00:00:00Z'"), sft)
```

Comment:
Nice!  What does the date "2016-12-131" do?!  Does it count forward 131 days?

Reply:
haha i have no idea... the test still seemed to pass :shrug:

Reply:
👀 😬